--- a/net/minecraft/world/entity/monster/Endermite.java
+++ b/net/minecraft/world/entity/monster/Endermite.java
@@ -89,6 +_,10 @@
    public void m_8119_() {
       this.f_20883_ = this.m_146908_();
       super.m_8119_();
+      if (!this.m_9236_().f_46443_ && this.m_21573_().m_26571_() && this.m_21563_().m_186069_()) {
+         this.m_146922_(this.f_20885_);
+         this.f_20883_ = this.f_20885_;
+      }
    }
 
    public void m_5618_(float p_32621_) {
